### DISCUSSION TITLE
Operating temperature of Raspberry Pi

### DISCUSSION THREAD

**Discussion by ondrejsevcik** [RaspAP/raspap-webgui#976] [Q&A] [answered]
I've noticed that my Pi is quite hot (~55℃) most of the time. I was wondering what is your operating temperature? Is it normal that it is that high? The RaspAP is the only sw running on my Pi, otherwise it's a stock (lite) raspbian. Also I don't have any case on my Raspberry.

**Comment by billz** [comment, marked as answer]
My RPi 4 runs 24/7 and tends to be in this range during the summer. Same software (RPi OS Lite + RaspAP), no case, flat orientation inside a glass cabinet with other hardware (router, NAS, etc). 

Pi's tend to run 20-30°C above the ambient temperature of the room/cabinet they are in, so this is within range. The services installed by RaspAP (dnsmasq, dhcpcd, hostapd, lighttpd) are all relatively lightweight in terms of CPU usage. Execute `top` and press `P` to sort by CPU.

**Reply by ondrejsevcik** [reply]
In that case all good :) Thank you for a quick answer.
